Day_03/Day_3.rb

```diff
@@ -0,0 +1,143 @@
+class HangmanGame
+  def initialize(total_words)
+    @total_words = total_words
+    @level = 1
+    @guessed_letters = []
+    @chances = 5
+  end
+
+  # Method to play the Hangman game at a specific level
+  def play_level
```

Comment:
one method should not be more than 10 lines of code so try to split this method in smaller methods